src/main/java/com/codecool/barbershop/barbershop/dashboard/DashboardController.java

```diff
@@ -0,0 +1,26 @@
+package com.codecool.barbershop.barbershop.dashboard;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping("api/v1/dashboard")
+public class DashboardController {
+    private final DashboardService dashboardService;
+
+    @Autowired
```

Comment:
Since you're injecting dependency through the constructor, you don't need the @Autowired anymore probably.